xls/dslx/bytecode/bytecode_interpreter_test.cc

```diff
@@ -1253,6 +1253,68 @@ fn main() -> MyU9 {
   EXPECT_THAT(value.GetBitValueUint64(), IsOkAndHolds(0x1ff));
 }
 
+TEST(BytecodeInterpreterTest, AssertEqFailProcs) {
+  constexpr std::string_view kProgram = R"(
+#[test_proc]
+proc BTester {
+    terminator: chan<bool> out;
+
+    init { (u32:0) }
+
+    config(terminator: chan<bool> out) {
+        (terminator,)
+    }
+
+    next(tok: token, state: u32) {
+        let count = state;
+        let _ = assert_eq(count, u32:0);
+        let tok = send_if(tok, terminator, count > u32:1, true);
+        (count + u32:1)
+    }
+})";
+
+  XLS_ASSERT_OK_AND_ASSIGN(auto temp_file,
+                           TempFile::CreateWithContent(kProgram, "_test.x"));
+  constexpr std::string_view kModuleName = "test";
+  ParseAndTestOptions options;
+  options.handle_status = false;
```

Comment:
couldn't find this field in the corresponding struct definition (am I missing something?)

Reply:
ah I see that's being added below. Can you document why the new field is necessary (either in the PR description or in the struct itself).

Reply:
I'll add comment to struct definition to explain existence of this field.